%YAML:1.0
---
camera_matrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 61226.148523221724, 0., 945.26011701170785, 0.,
       61143.196696286737, 497.58115877586391, 0., 0., 1. ]
distortion_coefficients: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ 7.460724105943747, 0.011472368881452302, 0.014729777934545298,
       0.083782527668016998, 1.8476156545649189e-05 ]
